models/classes/TestCategoryPreset.php

```diff
@@ -46,6 +46,11 @@ class TestCategoryPreset implements JsonSerializable
      */
     private $qtiCategory;
 
+    /**
+     * @var string[] $altCategories - the other possible qti categories that would activate the preset
+     */
+    private $altCategories;
```

Comment:
chore: There is no need to initialize it inside constructor https://github.com/oat-sa/extension-tao-testqti/pull/2461/files#diff-c5577ec1fd5d99becf2d82ed6ffc21e5f0cd54abfe2be7f978c690139257a473R100

```suggestion
    private $altCategories = [];
```

Reply:
Not necessary to refactor, but since PHP 7.4 we can declare properties with types.